oci/push.bzl

```diff
@@ -111,7 +160,28 @@ oci_push = rule(
         ),
         "tag": attr.string(
             doc = """
-                (optional) A tag to include in the target reference. This will not be included on child images."
+                (optional) A tag to include in the target reference. This will not be included on child images.
+
+                Subject to [$(location)](https://bazel.build/reference/be/make-variables#predefined_label_variables) and ["Make variable"](https://bazel.build/reference/be/make-variabmes) substitution.
+
+                **Stamping**
+
+                You can use values produced by the workspace status command in your tag. To do this write a script that prints key-value pairs separated by spaces, e.g.
+
+                ```sh
+                #!/usr/bin/env bash
+                echo "STABLE_KEY1 VALUE1"
+                echo "STABLE_KEY2 VALUE2"
+                ```
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** Not really something to block on, but as a heads up: it looks like these code blocks don't get rendered properly in the Markdown. I don't think it's worth blocking on because Markdown inside of tables likely requires an update to our dependencies that includes this change: https://github.com/bazelbuild/stardoc/pull/167.

Reply:
Nice! I can give upgrading a try in a separate PR if this is expected to work